corellium/adapter/src/test/kotlin/flank/corellium/adapter/AdaptersTest.kt

```diff
@@ -0,0 +1,72 @@
+package flank.corellium.adapter
+
+import flank.corellium.api.AndroidTestPlan
+import flank.corellium.api.Authorization
+import flank.corellium.api.invoke
+import flank.corellium.client.core.getAllProjects
+import flank.corellium.client.core.getProjectInstancesList
+import flank.corellium.corelliumApi
+import kotlinx.coroutines.flow.collect
+import kotlinx.coroutines.runBlocking
+
+private const val PROJECT_NAME = "Default Project"
+
+private val credentials = Authorization.Credentials(
+    host = TODO(),
```

Comment:
TODOS in a PR? 
Can we link them to tickets numbers if its not meant to be here to keep clarity.

Reply:
It's ok at this stage of that work in progress MVP.